app/controllers/carts_controller.rb

```diff
@@ -14,8 +14,28 @@ def checkout_cart
       flash[:alert] = 'You must login to checkout'
       redirect_to showcart_path
     else
-      redirect_to create_order_path
-    end
+      cart = Cart.new(session[:cart])
+      total_cost = 0
+      cart.items.each do |item, quantity|
+        total_cost += item.price * quantity.to_i
+      end
+
+      @order = Order.create(
+        user_id: session[:user_id],
+        total_cost: total_cost,
+        status: 'ordered'
+        )
+      cart.items.each do |item, quantity|
+        ItemOrder.create(
+          item_id: item.id,
+          order_id: @order.id,
+          quantity: quantity,
+          line_item_cost: item.price * quantity.to_i
+          )
+        end
```

Comment:
`end` at 35, 8 is not aligned with `cart.items.each do |item, quantity|` at 28, 6